modules/ai/assets/js/editor/utils/screenshot.js

```diff
@@ -1,6 +1,6 @@
 import { toCanvas } from 'html-to-image';
 import { toggleHistory } from './history';
-import { generateIds } from './generate-ids';
+import { generateIds } from '../context/requests-ids';
```

Comment:
Who is using the `generateIds`?

Reply:
```
function createContainer( template ) {
	const model = generateIds( template );

	// Set a custom ID, so it can be used later on in the backend.
	model.id = `e-ai-screenshot-container-${ model.id }`;

	return $e.run( 'document/elements/create', {
		container: elementor.getPreviewContainer(),
		model,
		options: {
			edit: false,
		},
	} );
}
```